Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", .exact(mapboxCommonVersion)),
+        .package(url: "https://github.com/mapbox/mapbox-common-ios.git", from: mapboxCommonVersion),
```

Comment:
According to SPM docs:
```
/// The following example allows the Swift Package Manager to select a version
/// like a  `1.2.3`, `1.2.4`, or `1.3.0`, but not `2.0.0`.
///
///    .package(url: "https://example.com/example-package.git", from: "1.2.3"),
```

Is it okay to allow minor versions in our cases(11.1.0 in this particular case can be installed)?